lib/cli/interactive-setup/utils.js

```diff
@@ -35,4 +38,12 @@ module.exports = {
 
     return Boolean(result);
   },
+  resolvePreliminaryContext: ({ configuration, serviceDir }) => {
+    return {
+      isInServiceContext: Boolean(serviceDir),
+      isLoggedIn: isAuthenticated(),
```

Comment:
And here maybe `isLoggedIntoDashboard` 

(as here we're not specifically in dashboard context, it might be valuable to indicate all dashboard specific properties with "dashboard" token)

Reply:
Good suggestion :+1: